adding_metadata_sql.md

```diff
@@ -0,0 +1,181 @@
+## SQL
+
+SQL can also be used for bulk uploads. However, SQL should primarily be used in the following cases:
+
+* fix data errors 
+* remove duplicate records
+* replace records
+* metadata upload requires features that are not available through the web interface or BETYdb-YABA
+
+
+### Metadata Entry Workflow for BETYdb
+
+The following steps have been implemented in the BETYdb-YABA API and python client, but are described below for reference
+
+Here we show SQL code used to add metadata required each season by the TERRA REF database, including experiments, sites, treatments, cultivars, and citations that is required prior to uploading the trait or yield data. Most PEcAn users _will not_ need to add experiments and cultivars, or associate these records with sites.
+
+Also note that in the TERRA REF and other agronomic applications, each record in the 'sites' table will correspond to an experimental plot.
+
```

Comment:
All the SQL statements are missing semicolons.

Reply:
fixed